homeassistant/components/zimi/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "zimi",
+  "name": "zimi",
+  "codeowners": ["@markhannon"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/zimi",
+  "iot_class": "local_push",
+  "requirements": ["zcc-helper==3.2.2"]
```

Comment:
we require libraries to be hosted on publicly accessible repositories (and have an OSI approved license), but I could only find you PyPi package, could you point me to it please